features/storage/kvstore/securestore/SecureStore.cpp

```diff
@@ -49,8 +49,6 @@ static const char *const auth_prefix = "AUTH";
 
 static const uint32_t security_flags = KVStore::REQUIRE_CONFIDENTIALITY_FLAG | KVStore::REQUIRE_REPLAY_PROTECTION_FLAG;
 
-namespace {
-
 typedef struct {
```

Comment:
Ah, the anonymous namespace was and still is required, if you have that `record_metadata_t` and `key_iterator_handle_t` outside your class. They collides with different definitions in other storage files. You'll need to move the `inc_set_handle_t` definition outside it.

(In C++ you're not allowed to have two definitions of the same type name. You normally get away with it, but LTO gets upset. An anonymous namespace is equivalent to `static`, but it works for types as well as objects.)

Reply:
Ah, true. Good catch.